tests/scenarios/scenarios.ts

```diff
@@ -35,13 +35,21 @@ async function release(project: Project) {
   project.linkDevDependency('ember-cli', { baseDir: __dirname, resolveName: 'ember-cli-latest' });
 }
 
+async function packager_webpack(project: Project) {
+  project.linkDevDependency('@embroider/webpack', { baseDir: __dirname });
+}
+
 export function supportMatrix(scenarios: Scenarios) {
-  return scenarios.expand({
-    lts_3_16,
-    lts_3_20,
-    lts_3_24,
-    release,
-  });
+  return scenarios
+    .expand({
+      lts_3_16,
+      lts_3_20,
+      lts_3_24,
+      release,
+    })
+    .expand({
```

Comment:
I think this is overkill and we should just add vite to this list:

```js
export function supportMatrix(scenarios: Scenarios) {
  return scenarios.expand({
    lts_3_16,
    lts_3_20,
    lts_3_24,
    release,
    vite
  });
```

The vite scenario would use the default ember version. If we later find some particular combinations that are problematic and need regression coverage, we can explicitly add those combinations.

Reply:
Oh okay, cool -- I can tweak it to work that way instead!

Should each of the "Ember Version" variants explicitly add the Embroider packager, and we'll remove the Webpack "variant" altogether?